edx_repo_tools/drip_survey/upload.py

```diff
@@ -0,0 +1,59 @@
+"""
+
+"""
+import csv
+from datetime import date
+import hashlib
+import sys
+
+import click
+from edx_repo_tools.data import pass_repo_tools_data
+
+NAME = 'FirstName'
+EMAIL = 'PrimaryEmail'
+WEEK = 'Week'
+ASSOCIATED_WITH = 'AssociatedWith'
+UNSUBSCRIBED = 'Unsubscribed'
+
+def associated_with(person):
```

Comment:
@cpennington: sure, provided you want to count people who sometimes contract with us as 'external'.

Weren't we going to put this kind of logic in its own package though?